app/src/main/java/com/monkeyteam/chimpagne/model/database/ChimpagneAccountManager.kt

```diff
@@ -72,12 +73,13 @@ class ChimpagneAccountManager(
       onSuccess: (ChimpagneAccount?, Uri?) -> Unit,
       onFailure: (Exception) -> Unit
   ) {
+
     getAccount(
         uid,
         { account ->
           if (account == null) onSuccess(null, null)
           else
-              downloadProfilePicture(
+              fetchProfilePictureUri(
```

Comment:
nice renamming, I was also confused at first with the name downloadProfilePicture.
This makes it clearers that we are storing the url and not the file itself